src/SeeMore/SeeMore.js

```diff
@@ -122,10 +124,13 @@ class SeeMore extends React.Component {
       <Text
         {...this.props}
         {...this.panResponder.panHandlers}
-        style={[style, { color: isLinkPressed ? linkPressedColor : linkColor }]}
-      >
+        style={[style, { color: isLinkPressed ? linkPressedColor : linkColor }]}>
         {isShowingMore ? null : <Text {...this.props}>...</Text>}
-        {isShowingMore ? ` ${seeLessText}` : ` ${seeMoreText}`}
+        {isShowingMore ? (
+          <Text style={seeLessTextStyle}>{seeLessText}</Text>
+        ) : (
+          <Text style={seeMoreTextStyle}>{seeMoreText}</Text>
```

Comment:
Seems like you have missed out on the space. If you notice the code before, I have mentioned the text as ` ${text}`

Reply:
Good catch, done ✅